src/qt/forms/createwalletdialog.ui

```diff
@@ -68,17 +71,30 @@
     <string>Encrypt Wallet</string>
    </property>
    <property name="checked">
-    <bool>true</bool>
+    <bool>false</bool>
+   </property>
+  </widget>
+  <widget class="QLabel" name="encrypt_wallet_label">
```

Comment:
why is the property with text "Advanced options" under a widget named 'encrypt_wallet_label' in the XML hierarchy? encrypting the wallet is just one option.

Reply:
That should have been `advanced_options_label`. I'll update that if I need to make another change.